shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -1511,11 +1511,90 @@ - (void)setEditableTransform:(NSArray*)matrix {
   _cachedFirstRect = kInvalidFirstRect;
 }
 
+- (BOOL)transformPoint:(CGPoint)point
+              isAffine:(BOOL)isAffine
```

Comment:
Could this be computed inside this function rather than passed in as a parameter? It's computed from `_editableTransform` and it doesn't look like we mutate that ivar in this method.